### PR TITLE
Feature/parallel slice

### PR DESCRIPTION
**Pull request by BaxterEaves**

### CODE REVIEW

**Review by schmidmt — APPROVED** [review]
Looks great! I'm glad it was so straightforward.

File: cli/Cargo.toml [hunk 1]
```diff
@@ -26,6 +26,3 @@ serde_yaml = "0.9.4"
 approx = "0.5.1"
 tempfile = "3.4"
 indoc = "2.0.3"
-
-[profile.release]
```
Comment (schmidmt) [inline cli/Cargo.toml:30]: Was there a reason for the removal?

**Review by Swandog — APPROVED** [review]
LGTM!

We'll have to remember to bump the `lace_utils` version before publishing, of course

File: cli/Cargo.toml (hunk 1 above)
Comment (BaxterEaves) [reply cli/Cargo.toml:30]: takes forever to compile

**Pull request merged**